### PR TITLE
Corrections resolving linting errors

### LINKED ISSUE #94: Corrections needed to resolve linting errors
**Problem description**
Spectral linter applied to device-status.yaml and produces 53 errors (!). Most of them are like:
```
  140:9     error  parser                          Mapping key must be a string scalar rather than number
                                                                           paths./connectivity.post.responses[200]
```

The response codes (200,...) should be strings
> This field MUST be enclosed in quotation marks (for example, "200") for compatibility between JSON and YAML.

Other errors are caused by numeric values of  event `id` in examples that should be quoted as stings:
```
  1008:13    error  oas3-valid-media-example        "id" property type must be string                                                                                                                      components.examples.ROAMING_STATUS.value.id
```

**Expected behavior**
Corrections for indicated errors

**Alternative solution**

**Additional context**
Linting rules are introduced in 
https://github.com/camaraproject/Commonalities/pull/74
https://github.com/camaraproject/Commonalities/pull/110

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:
Proposed changes correct errors detected by:

- Spectral (OAS linting tool):
    - error  parser                          Mapping key must be a string scalar rather than number
    - oas3-valid-schema-example       "example" property type must be string
    -  error  oas3-valid-media-example        "id" property type must be string
- yamllint:
    - [trailing-spaces] trailing spaces
    - [colons] too many spaces before colon
    - [brackets] too many spaces inside brackets

The version was changed to: `0.5.0-wip`

#### Which issue(s) this PR fixes:

Fixes #94 and #97 

#### Special notes for reviewers:

The output of linting can be seen after expanding "Megalinter" section of workflow results: 
https://github.com/rartych/DeviceStatus/actions/runs/7209051170/job/19639229860

#### Changelog input

```
Corrections resolving linting errors introduced

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by eric-murray — APPROVED** [review]

**Review by bigludo7 — APPROVED** [review]
LGTM

File: code/API_definitions/device-status.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.5.0-rc
+  version: 0.5.0-wip
```
Comment (fernandopradocabrillo) [inline code/API_definitions/device-status.yaml:102]: wip?

Comment (fernandopradocabrillo) [reply code/API_definitions/device-status.yaml:102]: I'm not familiar with the process, do we have to go back to wip and then make another PR for a release candidate?

Comment (rartych) [reply code/API_definitions/device-status.yaml:102]: Yes, the idea is to prepare RC2 after applying changes based on feedback from the latest RC, so the "wip" distinguishes fluid version with ongoing changes from RC.

**Comment by hdamker** [comment]
Added #95 as getting fixed by this PR to the description.

**Comment by akoshunyadi** [comment]
Correction for getting fixed by: #94 and #97

**Review by akoshunyadi — APPROVED** [review]

**Pull request merged**